### PR TITLE
Fix missing strings.

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**

### CODE REVIEW

**Comment by microbit-matt-hillsdon** [comment]
CI failure seems unrelated:

```
FAIL  src/e2e/edits.test.ts (15.474 s)
  ● Browser - edits › prompts on close if file edited

    expect(received).toEqual(expected) // deep equality

    Expected: true
    Received: false

      21 |     await app.findVisibleEditorContents(/A change/);
      22 |
    > 23 |     expect(await app.closePageCheckDialog()).toEqual(true);
         |                                              ^
      24 |   });
      25 |
      26 |   it("prompts on close if project name edited", async () => {

      at Object.<anonymous> (src/e2e/edits.test.ts:23:46)
```

File: lang/ui.en.json [hunk 1]
```diff
@@ -819,6 +819,14 @@
     "defaultMessage": "Hide simulator",
     "description": "Hide simulator action"
   },
+  "simulator-input-hold": {
+    "defaultMessage": "Hold",
+    "description": "Label for UI toggles to hold simulator buttons or bins"
```
Comment (microbit-robert) [inline lang/ui.en.json:824]:
```suggestion
    "description": "Label for UI toggles to hold simulator buttons or pins"
```

File: lang/ui.en.json [hunk 2]
```diff
@@ -819,6 +819,14 @@
...
+  },
+  "simulator-input-press": {
+    "defaultMessage": "Press",
+    "description": "Label for UI buttons to press simulator buttons or bins"
```
Comment (microbit-robert) [inline lang/ui.en.json:828]:
```suggestion
    "description": "Label for UI buttons to press simulator buttons or pins"
```

**Review by microbit-robert — CHANGES_REQUESTED** [review]
Small typo in string descriptions, otherwise looks good.

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Pull request merged**